yb-voyager/test/utils/cmdutils.go

```diff
@@ -34,13 +34,19 @@ func RunVoyagerCommand(container testcontainers.TestContainer,
 	cmdName string, cmdArgs []string, doDuringCmd func(), async bool) (*exec.Cmd, error) {
 
 	fmt.Printf("Running voyager command: %s %s\n", cmdName, strings.Join(cmdArgs, " "))
-	// Gather DB connection info.
-	host, port, err := container.GetHostPort()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get host port for container: %v", err)
-	}
+	var host string
+	var port int
+	var err error
+	var config testcontainers.ContainerConfig
+	if container != nil {
```

Comment:
why the change required here

Reply:
this was required as container is optional for some commands like `initiate cutover to target`

Reply:
Not planning to remove it in this PR as I still see some other usages to it in other tests